threema-connector-demo/webContent/layouts/basic-10.xhtml

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>

```

Comment:
also remove this one from the repo. we'll no longer consume it.

Reply:
removed